Add changelog and update Release.toml for v1.0.6

sanu11 opened the pull request:
<!--
Tips:
- Please read CONTRIBUTING.md to understand our process and our requests for PRs.
- Please file an issue before creating a PR so we can discuss the change and confirm it's not already being worked on.
-->

**Issue number:**
N/A


**Description of changes:**

Prepare the changelog for v1.0.6




**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

sanu11 committed:
Add changelog for v1.0.6

sanu11 commented on CHANGELOG.md:
```diff
@@ -1,3 +1,73 @@
+* Switch from TermLogger to SimpleLogger ([#1282], **thanks @hencrice!**)
+* Allow overriding the "pretty" name of the OS inside the image ([#1330])
+* Add aliases for container-selinux types ([#1316])
+* Declare dependencies on model and defaults files in storewolf ([#1319])
```
Should it be: ```storewolf: Declare dependencies on model and defaults files ``` ?

tjkirch replied:
Also, this can go under build changes.

sanu11 commented on CHANGELOG.md:
```diff
@@ -1,3 +1,73 @@
+* Add https-proxy and no-proxy settings to updog ([#1324])
+* Add support for ECR Public to hostctr ([#1296])
+* Add network proxy support to k8s 1.19 ([#1337])
+* Add support for kmot kit ([#1287], [#1286], [#1285])
```
Does this seem ok or sounds incomplete ?

tjkirch replied:
(typo: "kmod")

Maybe "Add support for kmod kits to ease building of third-party kernel modules"?

tjkirch replied:
Also, I think this can go under build changes.

tjkirch commented on CHANGELOG.md:
```diff
@@ -1,3 +1,73 @@
+# v1.0.6 (2021-02-26)
+
+## OS Changes
+* Add metricdog support to send anonoymous bottlerocket metrics ([#1006], [#1322])
```
```suggestion
* Add metricdog to support sending anonymous metrics ([#1006], [#1322])
```

tjkirch commented on CHANGELOG.md:
```diff
@@ -1,3 +1,73 @@
+## OS Changes
+* Add metricdog support to send anonoymous bottlerocket metrics ([#1006], [#1322])
+* Add a vmware-dev variant ([#1292], [#1288], [#1290])
+* Add k8s static pods support ([#1317])
```
```suggestion
* Add Kubernetes static pods support ([#1317])
```

tjkirch commented on CHANGELOG.md:
```diff
@@ -1,3 +1,73 @@
+* Add metricdog support to send anonoymous bottlerocket metrics ([#1006], [#1322])
+* Add a vmware-dev variant ([#1292], [#1288], [#1290])
+* Add k8s static pods support ([#1317])
+* Add high-level 'set' subcommand for changing settings in apiclient ([#1278])
```
```suggestion
* Add high-level 'set' subcommand for changing settings using apiclient ([#1278])
```

tjkirch commented on CHANGELOG.md:
```diff
@@ -1,3 +1,73 @@
+* Add a vmware-dev variant ([#1292], [#1288], [#1290])
+* Add k8s static pods support ([#1317])
+* Add high-level 'set' subcommand for changing settings in apiclient ([#1278])
+* Add "shibaken" generator to populate user-data with public keys from IMDS ([#1331])
```
```suggestion
* Allow admin container to use SSH public keys from user data ([#1331], [#19](https://github.com/bottlerocket-os/bottlerocket-admin-container/pull/19))
```

tjkirch commented on CHANGELOG.md:
```diff
@@ -1,3 +1,73 @@
+* Add "shibaken" generator to populate user-data with public keys from IMDS ([#1331])
+* Add support for kubelet in standalone mode and TLS auth ([#1338])
+* Add https-proxy and no-proxy settings to updog ([#1324])
+* Add support for ECR Public to hostctr ([#1296])
```
```suggestion
* Add support for pulling host-containers from ECR Public ([#1296])
```

tjkirch commented on CHANGELOG.md:
```diff
@@ -1,3 +1,73 @@
+* Add support for kubelet in standalone mode and TLS auth ([#1338])
+* Add https-proxy and no-proxy settings to updog ([#1324])
+* Add support for ECR Public to hostctr ([#1296])
+* Add network proxy support to k8s 1.19 ([#1337])
```
```suggestion
* Add network proxy support to aws-k8s-1.19 ([#1337])
```

tjkirch commented on CHANGELOG.md:
```diff
@@ -1,3 +1,73 @@
+* Add support for ECR Public to hostctr ([#1296])
+* Add network proxy support to k8s 1.19 ([#1337])
+* Add support for kmot kit ([#1287], [#1286], [#1285])
+* Modify default label for containers ([#1318])
```
```suggestion
* Modify default SELinux label for containers to align with upstream ([#1318])
```

tjkirch commented on CHANGELOG.md:
```diff
@@ -1,3 +1,73 @@
+* Add network proxy support to k8s 1.19 ([#1337])
+* Add support for kmot kit ([#1287], [#1286], [#1285])
+* Modify default label for containers ([#1318])
+* Switch from TermLogger to SimpleLogger ([#1282], **thanks @hencrice!**)
```
This can go under build changes, I think, it doesn't affect runtime.

tjkirch commented on CHANGELOG.md:
```diff
@@ -1,3 +1,73 @@
+* Add support for kmot kit ([#1287], [#1286], [#1285])
+* Modify default label for containers ([#1318])
+* Switch from TermLogger to SimpleLogger ([#1282], **thanks @hencrice!**)
+* Allow overriding the "pretty" name of the OS inside the image ([#1330])
```
This can go under build changes.

tjkirch commented on CHANGELOG.md:
```diff
@@ -1,3 +1,73 @@
+* Modify default label for containers ([#1318])
+* Switch from TermLogger to SimpleLogger ([#1282], **thanks @hencrice!**)
+* Allow overriding the "pretty" name of the OS inside the image ([#1330])
+* Add aliases for container-selinux types ([#1316])
```
```suggestion
* Add aliases for container-selinux types to align with community ([#1316])
```

tjkirch commented on CHANGELOG.md:
```diff
@@ -1,3 +1,73 @@
+* Allow overriding the "pretty" name of the OS inside the image ([#1330])
+* Add aliases for container-selinux types ([#1316])
+* Declare dependencies on model and defaults files in storewolf ([#1319])
+* Refactor default settings files to allow sharing in storewolf ([#1303])
```
```suggestion
* Refactor default settings files to allow sharing in storewolf ([#1303], [#1329])
```

Same "storewolf: " prefix as the prior line, if we do that.

Also, this can go under build changes.

tjkirch commented on CHANGELOG.md:
```diff
@@ -1,3 +1,73 @@
+* Add aliases for container-selinux types ([#1316])
+* Declare dependencies on model and defaults files in storewolf ([#1319])
+* Refactor default settings files to allow sharing in storewolf ([#1303])
+* Improve models symlink logic ([#1329])
```
```suggestion
```
This isn't a useful change on its own, it can be combined with 1303.

tjkirch commented on CHANGELOG.md:
```diff
@@ -1,3 +1,73 @@
+* Declare dependencies on model and defaults files in storewolf ([#1319])
+* Refactor default settings files to allow sharing in storewolf ([#1303])
+* Improve models symlink logic ([#1329])
+* Add the ability to output vmdk via qemu-img ([#1289])
```
This can go under build changes.

tjkirch commented on CHANGELOG.md:
```diff
@@ -1,3 +1,73 @@
+* Refactor default settings files to allow sharing in storewolf ([#1303])
+* Improve models symlink logic ([#1329])
+* Add the ability to output vmdk via qemu-img ([#1289])
+* Migrates default version of admin container from v0.5.2 to v0.6.0 ([#1347])
```
```suggestion
* Update default versions of admin and control containers ([#1347], [#1344])
```

And add https://github.com/bottlerocket-os/bottlerocket/pull/1344 to the link list below.

tjkirch commented on CHANGELOG.md:
```diff
@@ -1,3 +1,73 @@
+## Build Changes
+
+* Fix invalid symlinks when the BUILDSYS_NAME variable is set ([#1312])
+* Add script runner bash to link-variant ([#1323])
```
```suggestion
* Specify bash in link-variant task for use of bash features ([#1323])
```

sanu11 commented:
>  @sanu11 force-pushed the sanu11:develop branch from eb5cb98 to 4cd3db3 6 minutes ago 

Addressed comments by @tjkirch

tjkirch commented on CHANGELOG.md:
```diff
@@ -1,3 +1,74 @@
+# v1.0.6 (2021-02-26)
+
+## OS Changes
```
nit: for consistency, let's have a blank line under each heading, like past releases.

tjkirch reviewed (commented):
I think this looks good, but we need to include a couple more upcoming changes when they're ready.

webern commented on CHANGELOG.md:
```diff
@@ -1,3 +1,74 @@
+* Specify bash in link-variant task for use of bash features ([#1323])
+* Fix invalid symlinks when the BUILDSYS_NAME variable is set ([#1312])
+* Track and clean output files for builds ([#1291])
+* Update third-party software packages ([#1340], [#1336], [#1334], [#1333], [#1335], [#1190], [#1265], [#1315])
```
We can add #1352

sanu11 replied:
oh, its there in the recent force pushes, I think this one you are looking at is outdated ?

sanu11 commented:
> @sanu11 sanu11 force-pushed the sanu11:develop branch from 4cd3db3 to 8ed10ce 18 seconds ago

Added PRs #1352, #1353

tjkirch reviewed (commented):
Looks like we're missing #1327 in the OS changes section, and #1306 and #1339 in the documentation section, otherwise up to date.

sanu11 commented:
> Looks like we're missing #1327 in the OS changes section, and #1306 and #1339 in the documentation section, otherwise up to date.

oh sure. We weren't sure of 1306 and 1327 and thought they can be skipped thus didn't add them earlier. Will add them now : )

sanu11 commented:
> @sanu11 sanu11 force-pushed the sanu11:develop branch from 8ed10ce to 34cd784 11 seconds ago

Added  #1327, #1306 and #1339, changed release date and bumped release version

webern reviewed (changes requested):
~~Let's add the Release.toml version bump to this PR.~~

Edit: you just did!

sanu11 committed:
Update Release.toml for v1.0.6

sanu11 commented:
> @sanu11 sanu11 force-pushed the sanu11:develop branch from 34cd784 to 789b8b4 3 minutes ago

Modified #1306 content and moved Release.toml changes  to another commit
